packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -550,7 +551,10 @@ class _DraggableState<T extends Object> extends State<Draggable<T>> {
       axis: widget.axis,
       initialPosition: position,
       dragStartPoint: dragStartPoint,
-      feedback: widget.feedback,
+      feedback: DefaultTextStyle(
```

Comment:
In general we don't want to wrap these kind of widgets like that since the textstyle my not be needed at all or there could be many more inherited widgets that are needed by the widget. Instead, developers are encouraged to wrap the widget they pass to feedback with all the inherited widgets it needs. We should update the documentation to reflect that instead of making this change.

Reply:
OK I understood. Thanks for the feedback. 🙂